akka-docs/src/main/paradox/stream/operators/Sink/actorRefWithAck.md

```diff
@@ -1,4 +1,4 @@
-# actorRefWithAck
+# Sink.actorRefWithAck
```

Comment:
cc @raboof I think this is the style we should use in the headings, so it's obvious if a thing is only existing on one of the objects